package.json

```diff
@@ -419,6 +419,55 @@
           "type": "string",
           "description": "Name of vim function to call for choosing import candidates",
           "scope": "window"
+        },
+        "java.configuration.runtimes": {
```

Comment:
There are several new jdt.ls configuration options available now. Would it be preferred to add those in separate pull requests or all in one?